lib/job-runner/src/main.rs

```diff
@@ -16,6 +16,10 @@ const MAX_LINE_BYTES: usize = 1024;
 /// Maximum number of bytes to buffer before dropping logs
 const MAX_BUFFER_BYTES: usize = 1024 * 1024;
 
+/// Maximum number of lines to print to stdout for easy debugging why programs crash from Nomad's
+/// output.
+const MAX_PREVIEW_LINES: usize = 128;
```

Comment:
I found this like a bit hard to read. I think this is more clear:

```suggestion
/// Maximum number of lines to print to stdout for debugging. This helps
/// identify the reasons for program crashes based from Nomad's output.
const MAX_PREVIEW_LINES: usize = 128;
```